dll/win32/browseui/lang/de-DE.rc

```diff
@@ -31,7 +31,7 @@ BEGIN
         MENUITEM "S&ymbolleisten fixieren", IDM_TOOLBARS_LOCKTOOLBARS, MFT_STRING, MFS_ENABLED
         MENUITEM "&Anpassen...", IDM_TOOLBARS_CUSTOMIZE, MFT_STRING, MFS_ENABLED
         MENUITEM "Symboltitel", IDM_TOOLBARS_TEXTLABELS, MFT_STRING, MFS_ENABLED
-        MENUITEM "Schaltfläche Wechseln zu", IDM_TOOLBARS_GOBUTTON, MFT_STRING, MFS_ENABLED
+        MENUITEM "Schaltfläche wechseln zu", IDM_TOOLBARS_GOBUTTON, MFT_STRING, MFS_ENABLED
```

Comment:
Both translations are not clear, we discussed this in the chat just now, suggesting this translation:
```suggestion
        MENUITEM "Schaltfläche ""Wechseln zu""", IDM_TOOLBARS_GOBUTTON, MFT_STRING, MFS_ENABLED
```